waspc/src/Wasp/Project/Analyze.hs

```diff
@@ -74,7 +78,8 @@ constructAppSpec waspDir options decls = do
             AS.isBuild = CompileOptions.isBuild options,
             AS.userDockerfileContents = maybeUserDockerfileContents,
             AS.configFiles = configFiles,
-            AS.devDatabaseUrl = devDbUrl
+            AS.devDatabaseUrl = devDbUrl,
+            AS.staticClientAssetsDir = staticAssetsDir
```

Comment:
This is analogous to what we did for the`migrations` folder:
- check for the existence of the folder in `Analyse` module
- add info about the folder in `AppSpec`
- generate `copyDir` file draft in the `Generator` module